data/data/bootstrap/files/usr/local/bin/patch.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -x
+
+function patchit {
```

Comment:
Could naming be more explicit to usage? Maybe patch_etcd-quorum-guard

Reply:
if we omit quorum-guard with annotation will this function fail? I believe that was the intent behind "Add single-node-developer Cluster Profile" enhancement.[1]

[1]https://github.com/openshift/enhancements/pull/482

Reply:
Yes, this patch is just for allowing a single node installation and isn't related to the in-place-bootstrapping.